### PR TITLE
Upgrade Python to 3.7

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Upgrade runner's Python version to 3.7. 🙏 

### How to review 
Ensure:
- App behaviour is unchanged.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,15 +17,14 @@ WORKDIR /usr/src/app
...
-RUN ./scripts/translate_schemas.sh
 
 # We don't want node_modules to be copied to the runtime image
 RUN rm -rf node_modules
 ###############################################################################
 # Second Stage
 ###############################################################################
 
-FROM python:3.4-slim-stretch
+FROM python:3.7.2-slim-stretch
```
Comment (jonnyshaw89) [inline Dockerfile]: How about `python:3.7-slim-stretch` so we get patches for free?

File: app/forms/questionnaire_form.py [hunk 2]
```diff
@@ -254,7 +254,8 @@ def get_data(self, answer_id):
         return attr.raw_data[0] if attr.raw_data else ''
 
 
-def get_answer_fields(question, data, error_messages, schema, answer_store, metadata, group_instance, group_instance_id):
+def get_answer_fields(question, data, error_messages, schema, answer_store, metadata,  # pylint: disable=too-many-locals
```
Comment (jonnyshaw89) [inline app/forms/questionnaire_form.py]: this `pylint` should probably go on the next line as it breaks the parameter list

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: app/forms/questionnaire_form.py (hunk 2 above)
Comment (MebinAbraham) [reply app/forms/questionnaire_form.py]: Doesn't work on the next line, moved it to top.

File: Dockerfile [hunk 3]
```diff
@@ -17,15 +17,14 @@ WORKDIR /usr/src/app
 
 COPY . /usr/src/app
 RUN yarn compile
-RUN ./scripts/translate_schemas.sh
```
Comment (MebinAbraham) [inline Dockerfile:20]: Has been removed until the translation repo is ready.

File: Dockerfile [hunk 4]
```diff
@@ -36,12 +35,13 @@ WORKDIR /usr/src/app
 
 ENV GUNICORN_WORKERS 3
 
+COPY Pipfile Pipfile
 COPY Pipfile.lock Pipfile.lock
 COPY .python-version .python-version
```
Comment (bitdivision) [inline Dockerfile]: Can get rid of this?

**Comment by MebinAbraham** [comment]
I will try to sort the issue with Cypress as a separate PR as it's not related to this.

File: Dockerfile (hunk 4 above)
Comment (MebinAbraham) [reply Dockerfile]: Done

**Review by bitdivision — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**